crates/iota-sdk-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iota-sdk-types"
-version = "0.0.1"
+version = "0.0.1-alpha.0"
```

Comment:
Thinking out loud: is it common to have alpha before major 1 ? Since major 0 doesn't give any stability guarantee?

Reply:
I don't know. I don't really see the suffixes used that much tbh